MIGRATION.md

```diff
@@ -291,6 +291,7 @@ For our efforts to reduce bundle size of the SDK we had to remove and refactor p
 - Rename `UserAgent` integration to `HttpContext`. (see [#5027](https://github.com/getsentry/sentry-javascript/pull/5027))
 - Remove `SDK_NAME` export from `@sentry/browser`, `@sentry/node`, `@sentry/tracing` and `@sentry/vue` packages.
 - Removed `eventStatusFromHttpCode` to save on bundle size.
+- Replaced `Session` class with a session object and functions (see [#5054](https://github.com/getsentry/sentry-javascript/pull/5054)).
```

Comment:
Is this enough or do we need more information in Migration docs about the change?

Reply:
We'll need to add migration docs for Electron/React Native

Reply:
Added a section about the changes in https://github.com/getsentry/sentry-javascript/pull/5054/commits/5c81e32c45b2602ce1e65ec6b816c04ffdb31dc1